client/my-sites/upgrades/checkout-thank-you/header/index.jsx

```diff
@@ -4,18 +4,26 @@
 import classNames from 'classnames';
 import React from 'react';
 
+/**
+ * External dependencies
+ */
+import { isPlan } from 'lib/products-values';
+
 const CheckoutThankYouHeader = React.createClass( {
 	propTypes: {
 		isDataLoaded: React.PropTypes.bool.isRequired,
-		isFreeTrial: React.PropTypes.bool.isRequired,
-		productName: React.PropTypes.string
+		isFreeTrial: React.PropTypes.bool,
+		primaryPurchase: React.PropTypes.object,
+		selectedSite: React.PropTypes.object
 	},
 
 	renderHeading() {
```

Comment:
We should rename `renderHeading()` and `renderText()` into `getHeading()` and `getText()` since they just return a string.